generator-theia/src/extension/extension-package-generator.ts

```diff
@@ -72,19 +74,22 @@ export class ExtensionPackageGenerator extends AbstractGenerator {
                 "src"
             ],
             "scripts": {
-                "clean": "yarn run compile:clean",
+                "clean": "yarn run compile:clean && yarn run docs:clean",
                 "build": "concurrently -n compile,lint -c blue,green \"yarn run compile\" \"yarn run lint\"",
                 "compile": "tsc -p compile.tsconfig.json",
                 "compile:clean": "rimraf lib",
                 "lint": "tslint -c ../tslint.json --project compile.tsconfig.json",
                 "watch": "tsc -w -p compile.tsconfig.json",
+                "docs": "typedoc --tsconfig compile.tsconfig.json --options ../typedoc.json",
+                "docs:clean": "rimraf docs/api",
```

Comment:
You don't need --tsconfig I think it's just an alias to --options checj --helo

Reply:
I mean check --help

Reply:
I would like to avoid duplication of tsc settings, typedoc settings, with `--tsconfig compile.tsconfig.json` the package specific tsc settings are loaded and `--options ../typedoc.json` the common typedoc settings are loaded.

If `--tsconfig` would be just an alias of `--options` then it won't work.

Reply:
Are you sure it doesn't work without tsconfig ? Givem it does not use base i think. It may not do anything... Can't test myself right now

Reply:
I am not sure, feel free to fix it in this PR if it is doable.

Reply:
OK I was able to test and compile.tsconfig.json is beeing used and taking the extension: property into account.

So PR is ok with me. Please merge.